### DISCUSSION TITLE
trouble running with yarn

### DISCUSSION THREAD

**Discussion by dzgierski19** [scaffold-eth/scaffold-eth-2#605] [Q&A]
Hello, I have trouble running with yarn. Could u guys help me? I get this internal error
![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/106579123/00e0832e-178d-4757-af68-0aba8de06983)

**Comment by carletex** [comment]
Thanks for reporting this @dzgierski19!

Haven't seen that error before. Might be related to the new version of yarn? Your instance seems to be upgrading to 4.0.1 automatically, but SE2 should use yarn 3.2.3, which is included locally (inside /.yarn/releases/)

Could you run `yarn --version` inside the scaffold-eth-folder.

Could you also remove all node_modules folders (root, hardhat, nextjs), run `yarn cache clean` and then `yarn install` again... and see if it works?
